tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -14053,7 +14051,7 @@
   ],
   "css/border_black_groove.html": [
    "49e1647a6f71e320770225ad537b4fd4020bd700",
-   "reftest_node"
```

Comment:
I don't know what this means; I just ran update-manifest as usual and it changed this!

Reply:
probably something changed in the runner :man_shrugging: